src/modules/ekf2/ekf2_main.cpp

```diff
@@ -572,6 +628,13 @@ void Ekf2::run()
 {
 	bool imu_bias_reset_request = false;
 
+	// support for dual GPS inputs if available
+	int gps_subs[2];
```

Comment:
delete? _gps_subs in class

Reply:
I don't understand

Reply:
You have both gps_subs[2] and _gps_subs[ORB_MULTI_MAX_INSTANCES].

Reply:
I thought I needed a separate one for  publishing. will fix.